src/lib/notcurses.c

```diff
@@ -575,6 +575,10 @@ interrogate_terminfo(notcurses* nc, const notcurses_options* opts){
   char* longname_term = longname();
   fprintf(stderr, "Term: %s\n", longname_term ? longname_term : "?");
   nc->RGBflag = tigetflag("RGB") == 1;
+  if (nc->RGBflag == 0) {
+    const char *cterm = getenv("COLORTERM");
```

Comment:
nit: swap the * and the space. at minimum, a comment here would be nice.

is COLORTERM defined to anything besides "truecolor"/"24bit" that we might want to be aware of? we do quantization down to the palette if DirectColor isn't available.